vignettes/parallel.Rmd

```diff
@@ -13,12 +13,15 @@ vignette: >
   comment = "#>",
   out.width = "100%"
 )
+library(pizzarr)
 ```
 
 By default, reads and writes are performed sequentially (i.e., not in parallel).
 Users can opt-in to parallel read/write functionality via `options`.
 
-```{r}
+```{r, eval=FALSE}
```

Comment:
This pattern of installing is a little dubious, at least it shouldn't run when the vignette is run start to finish.

Reply:
Should we replace the `install.packages` with a `stop()` failure that suggests installation of the package?

Reply:
Probably. stop with a message saying the package should be installed is good, or if in interactive mode, it could prompt for user input?